src/content/manifest/hybrid-development.mdx

```diff
@@ -20,6 +20,8 @@ If you want to adopt this approach, you need to keep in mind that developing wit
 ### How hybrid development works?
 Usually, okteto allows you to develop a service locally and run it in a production-like cloud environment synchronizing the local changes with the development container running in the cloud (sync mode) but okteto also allows you to use another approach in which the service to be developed runs locally and the rest of the application components run in the cloud (hybrid mode).
```

Comment:
```suggestion
By default Okteto enables you to edit a service's code locally and have it run in a production-like cloud environment. This is achieved using `syncthing` where local changes are synchronized with the development container running in the cloud.

Okteto also allows you to use another mode called `hybrid` in which the service to be developed runs locally and the rest of the application components run in the cloud.
```

I'm wondering if we could just omit the first paragraph especially if it's mentioned elsewhere. It seems a bit odd to have a section header called "How does hybrid mode work" and then spent the first half talking about sync mode 🤔 

any opinions?

Reply:
agree

Reply:
check the preview after the changes in that section. Are you OK with it?